src/NServiceBus.Core/Recoverability/ProcessingFailureInfo.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using System.Runtime.ExceptionServices;
+
+    class ProcessingFailureInfo
+    {
+        public ProcessingFailureInfo(ExceptionDispatchInfo exceptionDispatchInfo, int flRetries, bool moveToErrorQueue = false, bool deferForSecondLevelRetry = false)
+        {
+            FLRetries = flRetries;
+            ExceptionDispatchInfo = exceptionDispatchInfo;
+            MoveToErrorQueue = moveToErrorQueue;
+            DeferForSecondLevelRetry = deferForSecondLevelRetry;
+        }
+        
+        public int FLRetries { get; }
```

Comment:
I think it would be better to not shortcut the name (same for the ctor argument)